### ISSUE TITLE
Apps/Games Crashing.

### ISSUE THREAD

**Issue by 1eta** [pwn20wndstuff/Undecimus#331]
I have the Uncover 3.0 and my iPad 2017 ios 11.4. Why do not many programs and games run: for example - Guitar pro, Monument valley 2, Space Age, Osmos, The Escapists 2 and many more. Very, very few programs and games running on this jailbreak. Why don't they work?

**Comment by Un4givenPG** [comment]
I’m guessing it’s jailbrek detection, you can do nothing about it, just restart and unjailbreak.

**Comment by 1eta** [comment]
I restarted it many times, it shows - Status: Re-Jailbreak. Jailbreak works very well, cydia works. I downloaded another game “the room three” and this game works, I don’t know why other games and programs don’t work, maybe there’s not enough optimization for them, I don’t know.

**Comment by king4q** [comment]
@1eta are they just crashing when you open them? Or do they just not load & stay on the loading screen?

**Comment by 1eta** [comment]
1

**Comment by Cryptiiiic** [comment]
@1eta did they work on previous betas? Maybe you have a tweak that you didn't have when it worked. Post your diagnostics.

**Comment by 1eta** [comment]
This happens on any version of uncover. I did not have tweaks when I  ran these programs. Today, I downloaded the programs/games from "Tongbu", and they work great! Why dont they work when I donwnload them from another resource, and install them manually ? I downloaded 5 programs/games, only 1 of 5 works "The Room three". :(
[diagnostics.zip](https://github.com/pwn20wndstuff/Undecimus/files/2846578/diagnostics.zip)
